fdbclient/Schemas.cpp

```diff
@@ -741,7 +741,9 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
          "auto_logs":3,
          "commit_proxies":5,
          "grv_proxies":1,
-         "backup_worker_enabled":1
+         "backup_worker_enabled":1,
+         "proxies":6,
+         "perpetual_storage_wiggle":0
```

Comment:
Was the `perpetual_storage_wiggle` change intended? If so, we also need to update `mr-status-json-schemas.rst.inc`

Reply:
It was not. That was the result of a bad merge somehow or other. Fixed.